src/test/scala/mesosphere/marathon/core/storage/zookeeper/ZooKeeperPersistenceStoreTest.scala

```diff
@@ -0,0 +1,382 @@
+package mesosphere.marathon
+package core.storage.zookeeper
+
+import akka.Done
+import akka.actor.ActorSystem
+import akka.stream.ActorMaterializer
+import akka.stream.scaladsl.Sink
+import akka.util.ByteString
+import com.typesafe.scalalogging.StrictLogging
+import mesosphere.UnitTest
+import mesosphere.marathon.core.storage.zookeeper.PersistenceStore._
+import mesosphere.marathon.util.ZookeeperServerTest
+import org.apache.curator.framework.CuratorFramework
+import org.apache.zookeeper.KeeperException.{NoNodeException, NodeExistsException}
+
+import scala.collection.concurrent.TrieMap
+import scala.concurrent.{Await, ExecutionContext}
+import scala.util.{Failure, Random, Success}
+
+class ZooKeeperPersistenceStoreTest extends UnitTest
```

Comment:
This seems to be an integration test since we use the ZooKeeper server.

Reply:
It has *neither* Mesos *nor* Marathon - only in-memory ZK. I'd say it's *not* an IT

Reply:
I agree.